### ISSUE TITLE
blank lines in bottom of console

### ISSUE THREAD

**Issue by pholck** [randy3k/radian#236]
Great tool; thanks much. 

Issue is r$> prompt is always about 6 lines from the bottom of my console screen (regardless if running radian stand-alone console, or running as R-interactive terminal inside VScode). That is, I always have "wasted screen space" at the bottom of my terminal/console, which is frustrating when using split screens etc in VScode. Couldn't find any options/settings to modify/correct. Any suggestions?

Thanks!

![image](https://user-images.githubusercontent.com/62806649/100668429-22719480-3319-11eb-9d03-68cc110c7100.png)

**Comment by randy3k** [comment]
The spaces are preserved for the completion menu. You could disable the auto completion while typing to regain the spaces, but obviously, you will need to trigger completions manually then.

**Comment by pholck** [comment]
Ahh...yes, that makes sense. Thank you!  Potential feature add would then
be to instead have the completion menu either go up or down, depending on
how space. Probably not an easy feature.

Thanks much, appreciate your work and help.

**Comment by adamaltmejd** [comment]
Was just going to ask about this. I guess its not possible for the completion menu to go up instead of down? Lots of valuable screen space lost otherwise. (otherwise really enjoying using radian, thank you!)

**Comment by randy3k** [comment]
You are right, the menu has to go down unless we change to a full screen app like vim.

**Comment by adamaltmejd** [comment]
Thanks! That sounds like a lot of work. I can survive without the menu on a small screen :)

**Comment by randy3k** [comment]
I also doubt if it is a good design choice to have a full screen app for repl.

**Comment by adamaltmejd** [comment]
Agreed.
